lib/authentication/view/edit_profile_page.dart

```diff
@@ -265,6 +333,10 @@ class _EditProfilePageState extends State<EditProfilePage> {
         child: Container(
           child: ListView(children: [
             AccountNotVerifiedWarning(),
+            Padding(
```

Comment:
After you rename the "loadImage" method like I said above, you can move the padding and gesture detector there and here just have `[AccountNotVerifiedWarning(), buildEditableAvatar(), PreferenceTitle...]`. Alternatively, you could even make the EditableAvatar a StatelessWidget and define it below.